src/pages/workspace/accounting/qbo/export/QuickbooksExportConfigurationPage.tsx

```diff
@@ -0,0 +1,105 @@
+    const sections = [
+        {
+            description: translate('workspace.qbo.preferredExporter'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_PREFERRED_EXPORTER.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.exporter),
+            title: exporter ?? policyOwner,
+        },
+        {
+            description: translate('workspace.qbo.date'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT_DATE_SELECT.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.exportDate),
+            title: exportDate ? translate(`workspace.qbo.${exportDate}.label`) : undefined,
+        },
+        {
+            description: translate('workspace.qbo.exportExpenses'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT_OUT_OF_POCKET_EXPENSES.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.syncCustomers),
+            title: exportEntity ? translate(`workspace.qbo.${exportEntity}`) : undefined,
+        },
+        {
+            description: translate('workspace.qbo.exportInvoices'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_INVOICE_ACCOUNT_SELECT.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.exportInvoice),
+            title: exportInvoice,
+        },
+        {
+            description: translate('workspace.qbo.exportCompany'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_COMPANY_CARD_EXPENSE.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.exportCompanyCard),
+            title: exportCompanyCard,
+        },
+        {
+            description: translate('workspace.qbo.exportExpensifyCard'),
+            action: () => Navigation.navigate(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_EXPORT.getRoute(policyID)),
+            hasError: Boolean(policy?.errors?.syncTaxes),
+            title: 'Credit card',
+            interactive: false,
+        },
+    ];
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            shouldEnableMaxHeight
+            testID={QuickbooksExportConfigurationPage.displayName}
+        >
+            <HeaderWithBackButton title={translate('workspace.qbo.export')} />
+            <ScrollView contentContainerStyle={styles.pb2}>
+                <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.qbo.exportDescription')}</Text>
+                {sections.map((section) => (
+                    <OfflineWithFeedback key={section.description}>
+                        <MenuItemWithTopDescription
+                            title={section.title}
+                            interactive={section?.interactive ?? true}
```

Comment:
@s77rt yes but we pass it from section - and it will be undefined - probably MenuItemWithTopDescription - pick this undefined and not use default 'true' value